### PR TITLE
Payload Receiver Service (Fluentd->FastAPI) v0 

### PR DESCRIPTION
**Pull request by sanjay920**

### CODE REVIEW

File: src/payload-receiver-service/Dockerfile
Comment (tybalex) [inline src/payload-receiver-service/Dockerfile:1]: a suggestion that apply to all of our images:
for images that the usage is as lightweight as this one, we should probably use python:3.8-slim instead.

**Review by tybalex — APPROVED** [review]
look good to me! just a minor suggestion as commented.
Let's wait for the PR #7 to be merged , and we can then merge this ! :)

**Review by AmartC — APPROVED** [review]
Looks good to me! We can merge this after the other PR is finalized.

File: src/payload-receiver-service/app/main.py
Comment (tybalex) [inline src/payload-receiver-service/app/main.py:20]: one more minor comments is, nw is already a global variable so we don't need to do `global nw` here.

File: src/payload-receiver-service/Dockerfile
Comment (sanjay920) [reply src/payload-receiver-service/Dockerfile:1]: using `python:3.8-slim ` now

File: src/payload-receiver-service/app/main.py
Comment (sanjay920) [reply src/payload-receiver-service/app/main.py:20]: `global nw` needs to be here because `@app.on_event("startup")` is a FastAPI related processes 

It can be removed from `get_nats()` though.

**Pull request merged**
